sdk_blueprints/Gretel_Advanced_Tabular_Blueprint.ipynb

```diff
@@ -0,0 +1 @@
iated section, as demonstrated below.\n","\n","- **Tip:** Use the `job_label` argument to append a descriptive label to the model's name."]},{"cell_type":"code","execution_count":null,"metadata":{"id":"105TCaRNFhRu"},"outputs":[],"source":["trained = gretel.submit_train(\n","    base_config=\"tabular-actgan\",\n","    data_source=dataset,\n","    job_label=\"custom-config\",\n","    params={\"epochs\": 800, \"discriminator_dim\": [1024, 1024, 1024]},\n","    privacy_filters={\"similarity\": \"high\", \"outliers\": None},\n",")"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"NYV5cC9NWZ2f"},"outputs":[],"source":["# view synthetic data quality scores\n","print(trained.report)"]},{"cell_type":"markdown","metadata":{"id":"FfWRuAFmlx-E"},"source":["## 🌱 Prepare the seed data\n","\n","- Conditional data generation is accomplished by submitting seed data, which can be given as a file path or `DataFrame`.\n","\n","- The seed data should contain a subset of the dataset's columns with the desired seed values.\n","\n","- Currently, only categorical seed columns are supported.\n","\n","- Here, we will conditionally generate `num_records` synthetic examples of high-school graduates making more than $50k per year."]},{"cell_type":"code","execution_count":null,"metadata":{"id":"vByXw-1il_T3"},"outputs":[],"source":["import pandas as pd\n","\n","num_records = 500\n","\n","seed_data = pd.DataFrame(\n","    {\n","        \"education\": [\"HS-grad\"] * num_records,\n","        \"income_bracket\": [\">50K\"] * num_records\n","    },\n",")\n","\n","seed_data"]},{"cell_type":"markdown","metadata":{"id":"l7v0K1APTclU"},"source":["## 🤖 Conditionally generate synthetic data\n","\n","- The `submit_generate` method requires either `num_records` **or** `seed_data` as a keyword argument.\n","\n","- If `seed_data` is given, the number of generated records will equal `len(seed_data)`.\n","\n","- **Tip:** You can generate data from any trained model in the current project by using its associated `model_id`."]},{"cell_type":"code","execution_count":null,"metadata":{"id":"DjxXnVSzWbLi"},"outputs":[],"source":["generated = gretel.submit_generate(trained.model_id, seed_data=seed_data)"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"R7xVnr-JU5Sd"},"outputs":[],"source":["# inspect conditionally generated data\n","generated.synthetic_data.head()"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"vda0uO1gmo2c"},"outputs":[],"source":["# verify that the seeded columns are correct\n","print(generated.synthetic_data[\"education\"].value_counts(), end=\"\\n\\n\")\n","print(generated.synthetic_data[\"income_bracket\"].value_counts())"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"7M2taxKRZXoH"},"outputs":[],"source":[]}],"metadata":{"colab":{"authorship_tag":"ABX9TyP0Au5grKrFRQXAP6neNTpF","provenance":[]},"kernelspec":{"display_name":"Python 3","name":"python3"},"language_info":{"name":"python"}},"nbformat":4,"nbformat_minor":0}
```

Comment:
Those markdown statements in the cell where you define the dataset path only get rendered in Colab. How about putting these in a separate markdown cell?